accounts/serializers.py

```diff
@@ -30,51 +47,79 @@ class Meta:
         )
 
 
-class UserSerializer(serializers.ModelSerializer):
+class UserSerializer(UUIDSerializerMixin, ModelSerializer):
     resource_name = 'users'
     url = serializers.HyperlinkedIdentityField(
         view_name='user-detail',
         lookup_field='uuid'
     )
 
-    included_serializers = {
-        'demographics': DemographicDataSerializer,
-    }
+    # included_serializers = {
+    #     'demographics': DemographicDataSerializer,
+    # }
+    demographics = UUIDResourceRelatedField(
+        queryset=DemographicData.objects,
+        many=True,
+        related_link_view_name='user-demographics-list',
+        related_link_url_kwarg='user_uuid',
+        related_link_lookup_field='uuid',
+    )
+    organization = UUIDResourceRelatedField(
+        queryset=Organization.objects,
+        many=False,
+        related_link_view_name='user-organizations-list',
+        related_link_url_kwarg='user_uuid',
+        related_link_lookup_field='uuid',
+    )
+    children = UUIDResourceRelatedField(
+        queryset=Child.objects,
+        many=True,
+        related_link_view_name='user-children-list',
+        related_link_url_kwarg='user_uuid',
+        related_link_lookup_field='uuid',
+    )
 
     class Meta:
         model = User
         fields = (
             'url',
-            'uuid',
             'given_name',
             'middle_name',
             'family_name',
             'identicon',
             'is_active',
             'is_staff',
-            'demographics'
+            'demographics',
+            'organization',
+            'children'
         )
 
-    class JSONAPIMeta:
-        included_resources = ['demographics', ]
+    # class JSONAPIMeta:
+    #     included_resources = ['demographics', ]
 
 
-class ChildSerializer(serializers.ModelSerializer):
+class ChildSerializer(UUIDSerializerMixin, ModelSerializer):
     lookup_field = 'uuid'
     url = serializers.HyperlinkedIdentityField(
         view_name='child-detail',
         lookup_field='uuid'
     )
-    included_serializers = {
-        'user': UserSerializer,
-    }
+    # included_serializers = {
+    #     'user': UserSerializer,
+    # }
+    user = UUIDResourceRelatedField(
+        queryset=User.objects,
+        many=False,
+        related_link_view_name='child-users-list',
```

Comment:
I can probably do `users/user_id` for all of these if that works for you. I think it makes more sense anyway since there's only one.